frontend/ui/data/creoptions.lua

```diff
@@ -404,7 +404,7 @@ Note that your selected font size is not affected by this setting.]]),
         options = {
             {
                 name = "font_size",
-                alt_name_text = _("Font size"),
+                alt_name_text = _("Font Size"),
```

Comment:
(Just asking: when do we see this one ? I can't manage to get to see it...)

Reply:
Bottom menu Aa - hold the font size numbers.

`alt_name_text` was introduced as a fix for https://github.com/koreader/koreader/issues/7231.
It is not used anywhere besides this dialog. It can be changed to usual `name_text` here too.

Reply:
I generally have a preference toward less of the caps style. This is in a context where they're all using it?

Reply:
4 `Font Size` entries:
- bottom menu item (all items starting with caps)
- title of the spinwdget
- text in confirmbox "Set as default" in spinwidget
- text in confirmbox after holding the fontsize numbers (my case)

Reply:
+ one more: infomessage when holding Font Size in the bottom menu itself

Reply:
Other uses of the string "font size" aren't context. ;-)

> text in confirmbox "Set as default" in spinwidget

Should be `Font size`, ftr.

Reply:
context: all the bottom menu stuff handling via the configdialog default handlers use the names we set in creoptions/koptions, and all these (except this font size one what is being corrected) are "All Words Caps" (which even if it looks odd, fixes the consistency :)

Reply:
Indeed, I found it now. :-)